pkg/skaffold/build/dependencies_test.go

```diff
@@ -58,12 +55,12 @@ type mockArtifactResolver struct {
 	m map[string]string
 }
```

Comment:
Given the new bool, I think you could simplify a bit and get rid of the failingArtifactResolver with something like:
```
type mockArtifactResolver map[string]string
func (r mockArtifactResolver) GetImageTag(imageName string) (string, bool) {
	tag, found := r[imageName]
	return tag, found
}
```